main.go

```diff
@@ -19,8 +19,15 @@ import (
 	"github.com/nats-io/nsc/cmd"
 )
 
-// the cli injects the version
-var version = "0.0.0-dev"
+// The build system should inject this data.
+// We only currently use the version, because of cobra's model, but we stamp
+// the rest into the binary so that debuggers can see it.
+var (
```

Comment:
linter is failing the tests based on this.

Reply:
Fixed, thanks.